HTML-conversions/Save-image-in-folder-in-Word-to-HTML/.NET-Standard/Save-image-in-folder-in-Word-to-HTML/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.IO;
+
+namespace Save_image_in_folder_in_Word_to_HTML
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Load an existing Word document. 
+            using (FileStream fileStreamPath = new FileStream(Path.GetFullPath(@"../../../Input.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            {
+                //Load the file stream into a Word document.
+                using (WordDocument document = new WordDocument(fileStreamPath, FormatType.Docx))
+                {
+                    //Create a file stream.
+                    using (FileStream outputFileStream = new FileStream(Path.GetFullPath(@"../../../WordToHtml.html"), FileMode.Create, FileAccess.ReadWrite))
+                    {
+                        //Hook the event to customize the image. 
+                        document.SaveOptions.ImageNodeVisited += SaveImage;
+                        //Save the html file to the file stream.
+                        document.Save(outputFileStream, FormatType.Html);
+                       
+                    }                                      
+                }
+            }
+        }
+        //The following code examples show the event handler to customize the image path and save the image in an external folder.
+        static void SaveImage(object sender, ImageNodeVisitedEventArgs args)
+        {
+            string imagepath = @"D:\Temp\Image1.png";
```

Comment:
This code is not generic. If your Word document contains multiple images, all the images saves as same name and the last image only used in the HTML

Use generic solution for image path. Then only all the images are saved as new name and all those images are used in the HTML.